### ISSUE TITLE
Pi Node rewards turn to 0

### ISSUE THREAD

**Issue by JengOuyang** [pi-node/instructions#283]
My Pi nodes rewards has turn to "0" for several days, but the Pi Node status seems work well. 
Any one have same issue and how you solve this issue?
![Node Status](https://user-images.githubusercontent.com/97074982/176877227-a1ddd574-53b5-49bb-9561-37785dac584c.PNG)
![Rewards](https://user-images.githubusercontent.com/97074982/176877229-eba2d575-792a-4316-9280-f9d29e2f23d3.PNG)

**Comment by michelferia** [comment]
Seems that is giving rewards today, I had the same issue but it is ok now, check to see if your node is good now

**Comment by michelferia** [comment]
Well, righ now rewards is 0 and my node is synced!
any idea? please let me know if the rewards are gone forever to shutdown the Node ASAP...
I would like to know if someone will run a node (expending time, electricity, hardware, etc) without rewards :)

thanks!

**Comment by NeyMR** [comment]
3 days straight same issue no bonuses 0.0, and no access to node chat on the Pi App we need some clarification on what’s going on we are running blind at this time 😞, thank you in advance @MastaPyCharm

**Comment by sureshjkb** [comment]
5 days no teward bonus for pinode testnet than 1 day got and again from last two days bonus is zero despite pi testnode running perfectly with all required parameters as ok. Can someone guide what to do

**Comment by rezoan** [comment]
I am running my node on an azure VM, still no reward. very strange...

**Comment by xen888** [comment]
Node bonus reward is still 0 for 4 days....

**Comment by cheme75** [comment]
Mine has been dropping to 0 reward randomly for at least 2 months now, and every time I look, node is running fine. Normally it only drops for one day or two, but this time it’s been down to 0 all week. It’s hard to justify the effort and expense to keep it running with so many 0 days and not one word from the core team. They seem to live in their own little world and throw us some bread crumbs to keep our interest going. The long this festers, the more it stinks of an elaborate scam.

**Comment by NeyMR** [comment]
Yes as well experiencing 0.0 Node bonuses for about 4 days but we tend to forget we're all in testnet and with any testing downtimes and adjusting will occur, lets give CT time to resolve many are reporting Bonuses are back on there Nodes unfortunately my node is still under tuning just be patient your node chain will be updated soon and bonuses will be back..give the team time..

**Comment by michelferia** [comment]
![image](https://user-images.githubusercontent.com/11069310/180650758-feff3448-8a57-4170-ab29-89af26419d52.png)
![image](https://user-images.githubusercontent.com/11069310/180650766-2ff2da8e-b94c-41f5-b5c7-e34a3225a83a.png)

My pi node is perfectly working right now and no node rewards since some days
I need the confirmation that no rewards will distributed to stop my node asap, please someone of the CT could check this thread?
thanks!

**Comment by cheme75** [comment]
Fwiw, my node returned this morning when I re-started mining. I did reboot my pc on Friday due to a power outage. 

Btw, don’t hold your breath waiting for CT replies, they have yet to respond to the many times users have pointed out security holes exist in various library files they are using. Have yet to see a single patch or update since I started the node.

**Comment by NeyMR** [comment]
Utility usage bonus
Tuning in progress. This value may change
Node Bonus
Tuning in progress. This value may change
written in red ??? .... 
again And for information about bonuses- in the white papper  formula the Availability (Uptime), Ports Open, and CPUs factors are all added together then multiplied by the Node Uptime from the previous day. If the Node was inactive then its uptime is 0 and 0 * Anything is 0. There is no passive reward for past activity alone. Same applies to the App usage reward, if you don't use apps the day before then no reward on the current day…

and for many who don't browse the pinned messages Dr nicolas explained briefly about is occurring and its normal 
"@nicolas
@huuprhuoc2091 nothing wrong with your node. We are just slowly and carefully scaling up, while running multiple experiments. For example, we recently had our first tesnet reset. As a reminder the Testnet is supposed to reset at least once a quarter and after major tests."

so I can understand that many feel confortable with the bonus for testing and shocked when is not available but it is a test and guaranteed your node is going to go through evaluation periods etc where bonus might be redacted for evaluation purposes to make sure all nodes are properly being calculated and evaluated, please lets show patience, we're all in this together😁👍

best regards
@MastaPyCharm

**Comment by cheme75** [comment]
I can see a session at zero if my node happens to be down when mining session is started. But a 20 min power outage should only affect the uptime part of the calculation. Overall its around 90% up and I do see the bonus vary, which is certainly understandable. However, when the node downtime is only 20 min, but bonus disappears for 4-5 days starting before the pc even went down or when the pc never even went down, then those extended durations withholding the node bonus should be at least posted - here, twitter, etc. - for our knowledge of the fact it’s a testing issue. They should create a chat room for node operators if they don’t want to post here or on twitter.

**Comment by xen888** [comment]
0 node bonus. CT violating their white paper themselves.

**Comment by NeyMR** [comment]
> 0 node bonus. CT violating their white paper themselves.

you must explain your accusation basis how are they breaking their own WP? when is clearly your not understanding your role as a tester on a testnet🤔

**Comment by incarvr6** [comment]
If there is 0 node bonus what is the point of running it???
If it would be on something like PP4 then yes i don't mind using 2.5W but as many running it on PC with 25W and more that is wrong....
25W a year that is £76.65 @0.35pKW

**Comment by NeyMR** [comment]
please is completely normal your  node quorum may go under tuning and its not just your node, we have all experienced and will continue to experience tuning periods while we're all on the testnet, you fail to see how the calculations or even our nodes is calculated in the halving etc all have to be adjusted it is well explained but most just understand one part and fail to understand the entirety of ecosystem, have patience and please trust in PCT they are doing a great job 👏, we're all in this together 😁👍

**Comment by incarvr6** [comment]
(fail to see how the calculations or even our nodes is calculated) !!!
Yes absolutely because it is not documented anywhare!  But it should't be 0 for more that two weeks for node with 99.2% reliability!!! And it is similar or same with my friends nodes all have 0!!!!

**Comment by cheme75** [comment]
Fwiw, I got my node bonus back this week / mostly, had one day at zero. Better than had been previous week.

**Comment by NeyMR** [comment]
> (fail to see how the calculations or even our nodes is calculated) !!!
> 
> Yes absolutely because it is not documented anywhare!  But it should't be 0 for more that two weeks for node with 99.2% reliability!!! And it is similar or same with my friends nodes all have 0!!!!

is very well documented is sometimes we fail to understand is ok we're all in this together 👇👇

Dear pioneers ☝️☝️☝️ Node bonuses - ⚠️Again - development , tuning…. and please read also NODE FAQ about rewards for node in testnet ( we all still sometimes get it and sometimes no - tuning) minepi.com/node-info

Will There Be A Reward For Running A Pi Node?
One of the goals of Pi’s Testnet is to determine what, if any, rewards there should be for running a Node. While we collect the relevant data needed to determine the most appropriate incentive mechanism, there will be no mining rewards for Testnet Nodes. …. 
Thanks for your understanding patience and best regards  ps. Your date of installation? I’m running node since March 2020 and if you install node just because of bonuses, sorry, I think you read to little…. Best regards 

@MastaPyCharm I stand by The PCT lets give them support not unfounded flames, we're node operators the more advanced contribution to the ecosystem we're all in this together 😁👍

**Comment by cheme75** [comment]
Fwiw, node has been back online all week after a couple of outages. Finally today, after a week back online I just got mode bonus today. Absurd to take a week to catch-up. But still no kyc invite so it all vapor at this point. Plus 90% of my earnings came from those no longer active and most like have walked away so doubtful I’ll have much value to transfer if I ever get the kyc.

**Comment by NeyMR** [comment]
made a telegram for node only help and talk all welcome to join https://t.me/+fg3NodtmBpM5ZmJh

⚠️Any KYC mentions will be muted⚠️ just an heads up trying to keep it on topic and clean 😁👍

we can help each other faster with questions and issues, we're all in this together 😁👍

**Comment by cheme75** [comment]
Not interested in telegram. Got too many apps I’m essentially forced into using like discord just for the scrypted dev. But at least he is actively supporting users and responds to requests. Without the Pi node devs involvement, I’ll pass.

**Comment by NeyMR** [comment]
> Not interested in telegram. Got too many apps I’m essentially forced into using like discord just for the scrypted dev. But at least he is actively supporting users and responds to requests. Without the Pi node devs involvement, I’ll pass. 

completely understand, but is always good to have resources as many of us do not experience the issues many encounter and thats may be to knowledge 🤔 that we're willing to share 🤓👍, the frustration of many comes in the knowledge, not that your don't have knowledge in this matter, as I know scriptdev well, back in my mongodb deving days interacted in there forums a lot with running my private gaming server but that was a while back, anyhow, is always good to have more hands on deck that we can rely on, thanks for your understanding and support, we're all in this together 😁👍

**Comment by cheme75** [comment]
If any official pi representatives show up, be sure to post here - I’ll definitely join.

**Comment by amul69906995** [comment]
my node bouns goes up for 3 days then reset to 0 then start again
